chronos/ChronosAutoImport/main.py

```diff
@@ -0,0 +1,76 @@
+import os
+import requests
+
+from getpass import getpass
+from mysql.connector import connect, Error
+
+
+CHRONOS_LOCATION = "C:/ProgramData/MySQL/MySQL Server 8.0/Uploads/"
+CHRONOS_NAME = "chronos.csv"
+SCRIPT_PATH = "C:\\Theseus\\Tools\\dbscripts\\armory_fast_import_min.sql"
```

Comment:
Commit this file too. Also forward slashes, like for `CHRONOS_LOCATION`.

Reply:
Or instead of a manually minified file, this should work:
```py
with open("armory_fast_import.sql", "r") as fp:
    lines = fp.readlines()

lines = [line.strip() for line in lines if not line.strip().startswith("#")]
lines = " ".join(lines)

>>> 'TRUNCATE item_list; LOAD DATA INFILE \'C:/ProgramData/MySQL/MySQL Server 8.0/Uploads/chronos.csv\' INTO TABLE item_list FIELDS TERMINATED BY \',\' ENCLOSED BY \'"\' LINES TERMINATED BY \'\\r\\n\' IGNORE 1 LINES (className, prettyName, category, subCategory, slot, price, salePrice, description, supplier, imageURL, available, accessLevel, requiredTrainings, availableMembers);'
```

Reply:
It works, and it's implemented now.